src/silicone/database_crunchers/quantile_rolling_windows.py

```diff
@@ -211,6 +211,7 @@ def derive_relationship(
                 # We must prevent singularity behaviour if all the points are at the
                 # same x value.
                 cumsum_weights = np.array([(0.5 + x) / len(ys) for x in range(len(ys))])
+                ys.sort()
```

Comment:
```suggestion
```

Reply:
I need this sort for the interpolate limits.

Reply:
does it do it in place? my bad